src/java/ognl/security/OgnlPolicy.java

```diff
@@ -0,0 +1,56 @@
+package ognl.security;
+
+import java.lang.reflect.ReflectPermission;
```

Comment:
This may be an unused import (IDE reported)

Reply:
What's your IDE? It wasn't unused. I had `...instance of...` for that.

Reply:
I'm using NetBeans 8 currently.  The import does appear to be unused as `ReflectPermission` only appears in the import statement itself ?

Commenting the import out, it still compiled for me (with my slightly-out-of-date local version).  Looks like `ReflectPermission` only appears in the import in the latest revision too.

Reply:
Oh sorry you were right. I confused it with OgnlSecurityManager file :(